website/Taskfile.yml

```diff
@@ -33,20 +39,15 @@ tasks:
       HOST: '{{default "localhost" .HOST}}'
       PORT: '{{default "3001" .PORT}}'
     cmds:
-      - npx docusaurus serve --no-open --host={{.HOST}} --port={{.PORT}}
+      - pnpm preview  --host={{.HOST}} --port={{.PORT}}
 
   clean:
     desc: Clean temp directories
     cmds:
-      - rm -rf ./build
+      - rm -rf ./vitepress/dist
 
   deploy:
-    desc: Build and deploy Docusaurus
-    summary: Requires GIT_USER and GIT_PASS envs to be previous set
-    cmds:
-      - npx docusaurus deploy
-
-  upgrade:
-    desc: Upgrade Docusaurus
+    deps: [build]
+    desc: Build and deploy Website to Netlify
     cmds:
-      - yarn upgrade @docusaurus/core@latest @docusaurus/preset-classic@latest @docusaurus/module-type-aliases@latest @docusaurus/tsconfig@latest @docusaurus/types@latest
+      - pnpm netlify deploy --prod --site=054cd897-66d2-46fa-9b57-4831a8d61b37
```

Comment:
@vmaerten Which site ID is this? I suppose it's a test one you created.

We're probably going to remove this because we'll have two sites now: `taskfile.dev` and `next.taskfile.dev`. We should set this via ENV.

Basically we'll need to:

* Create a new website on Netlify to be `next.taskfile.dev` or change the domain of the current one to be `next.taskfile.dev`. IIRC it's Netlify that receives webhooks for pushes to `main` and builds the site.
* Set up the website that will be the new `taskfile.dev` to deploy from GitHub Action on release as you did.

I can work on this if you want.

Reply:
Exactly, I planned to change it as soon we agree on the PR and netlify has been updated

We don't need to variabilise it because only `taskfile.dev` will be deployed with this Task
`next.taskfile.dev` will be deployed "from Github" like we have today
So this siteId should be the `taskfile.dev`'s one

> I can work on this if you want.

As you wish, if you have time, I am fine with it :)